docs/core/install/includes/linux-install-manual.md

```diff
@@ -0,0 +1,33 @@
+
+Both .NET Core SDK and .NET Core Runtime can be manually installed after they've been downloaded. First, download a binary release for either the SDK or the runtime from one of the following sites:
```

Comment:
If you install the SDK, do you need to install the Runtime? I think we should be more explicit about when you need to do one versus the other. For example, the Runtime is usually installed on target environments where the SDK is not needed. The SDK is usually installed by the developers, and it includes the Runtime.